examples/scripts/making_a_bot.py

```diff
@@ -53,7 +53,7 @@ def run(self) -> None:  # overwrite the AbstractBotJob run method
                 self.save(lambda: edition.save(comment=comment))
 
 
-if '__name__' == __name__:
+if __name__ == "__main__ ":
```

Comment:
```suggestion
if __name__ == "__main__":
```
Space characters are significant.